src/header&footer/MyFooter.jsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+import styles from './header&footer.scss';
```

Comment:
you can have separate style files for header & footer since they are separate components